### PR TITLE
Added activate call

### PR DESCRIPTION
**Pull request by digitaldjango**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,15 @@ A webhook is exposed at POST /optimizely/webhook to be called by Optimizely when
...
 # Caching
 The activate() call is cached to reduce usage of Optimizely impressions.
+
+# Basic usage in TemperWeb
+```
+use Optimizely
+
+...
+
+$variant = Optimizely->activate($experimentName, Auth::user()->getRouteKey(), $params);
```
Comment (Robertbaelde) [inline README.md]: Must be `Optimizely::activate(...`

File: src/OptimizelyWrapper.php [hunk 2]
```diff
@@ -5,30 +5,56 @@
...
         $this->datafile = app()->make(Datafile::class)->get();
         $this->optimizely = app()->make(Optimizely::class);
 
-        $cacheKey = 'isFeatureEnabled-' . $experiment . $userID . md5(json_encode($params)) . md5($this->datafile);
+        $cacheKey = $type . '-' . $experiment . $userID . md5(json_encode($params)) . md5($this->datafile);
 
         if (!Cache::has($cacheKey))
         {
-            $variant = $this->optimizely->isFeatureEnabled($experiment, $userID, $params);
+            if ($type === self::TYPE_IS_FEATURE_ENABLED) {
```
Comment (Robertbaelde) [inline src/OptimizelyWrapper.php]: You could do 
`$variant = $this->optimizely->$type($experiment, $userId, $params);` here

**Review by Robertbaelde — CHANGES_REQUESTED** [review]

File: src/OptimizelyWrapper.php (hunk 2 above)
Comment (JeroenJochems) [reply src/OptimizelyWrapper.php]: 👍

Comment (JeroenJochems) [reply src/OptimizelyWrapper.php]: I think it's good enough. Feel free to merge at will @Speedrockracer, or adjust according to @Robertbaelde's review.

**Review by JeroenJochems — APPROVED** [review]

**Comment by digitaldjango** [comment]
I'll merge it tomorrow when I have some time to test if everything is working as expected :)

**Review by Robertbaelde — APPROVED** [review]

**Review by YordiLorenzo — CHANGES_REQUESTED** [review]
@Speedrockracer why are we using A/B experiments instead of a feature test? Feature tests are the preferred way of working.

**Pull request merged**